software/apps/audio_module/event_detector/README.md

```diff
@@ -0,0 +1,14 @@
+# Fixed-point ridge tracker for acoustic event detection in C
+
+## Quick start
+```
+make clean
+make
+./main input.wav
+```
+Use test.ipynb to visualize the resulting \*.mat files
+
+## Prerequisites
+* Fixed-point FFT: git clone https://github.com/longle2718/kiss_fft
```

Comment:
I agree. However, once we have a hierarchical directory structure, we won't be copying the sources and just leave them as prerequisites. Thanks!